apps/meteor/app/apps/server/bridges/rooms.ts

```diff
@@ -163,13 +163,13 @@ export class AppRoomBridge extends RoomBridge {
 	protected async update(room: IRoom, members: Array<string> = [], appId: string): Promise<void> {
 		this.orch.debugLog(`The App ${appId} is updating a room.`);
 
-		if (!room.id || !(await Rooms.findOneById(room.id))) {
-			throw new Error('A room must exist to update.');
-		}
+		const rm = await this.orch.getConverters()?.get('rooms').convertAppRoom(room, true);
 
-		const rm = await this.orch.getConverters()?.get('rooms').convertAppRoom(room);
+		const updateResult = await Rooms.updateOne({ _id: room.id }, { $set: rm });
```

Comment:
btw, since we are doing manual notifications, should we notify that the room changed here? with `notifyRoomChanged`?

Reply:
hmmm maybe? I think that deserves a different PR, though. To be honest, I'm unsure why we're doing the update manually